packages/dai/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@makerdao/dai",
+  "version": "0.16.2",
+  "contributors": [
+    "Wouter Kampmann <wouter@makerdao.com>",
+    "Sean Brennan <sean@makerdao.com>",
+    "Tyler Sorensen <tyler@makerdao.com>",
+    "Ethan Bennett <ethan@makerdao.com>",
+    "Lawrence Wang <lawrence@makerdao.com>",
+    "Joshua Levine <joshua@makerdao.com>",
+    "Michael Elliot <mike@makerdao.com>"
+  ],
+  "description": "Library for interacting with the Dai Stablecoin System.",
+  "license": "MIT",
+  "keywords": [
+    "makerdao",
+    "mkr",
+    "dai",
+    "cdp",
+    "eth",
+    "margin"
+  ],
+  "bugs": {
+    "url": "https://github.com/makerdao/dai.js/issues",
+    "email": "wouter@makerdao.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/makerdao/dai.js.git"
+  },
+  "module": "dist/src/index.js",
+  "main": "dist/src/index.js",
```

Comment:
Important to note. This change means that we now want to run `yarn link`/`yalc` in `packages/dai` NOT `packages/dai/dist`.